src/common/hooks/useIdentifierResolver.ts

```diff
@@ -8,8 +8,6 @@ export const useIdentifierResolver = (address: string) => {
   const { thirdPartyAPIEndpoints } = useThirdPartyAPIEndpoints();
   const { getIdentifier } = useAddressBook();
 
-  useEffect(() => setResolvedIdentifier(""), [address]); // unset resolvedIdentifier when address is different
-
   useEffect(() => {
     if (address === "") return;
```

Comment:
is this really ok to prevent this code to run ? what about null/undefined (I know typescript could raise error in this case but it's easy to bypass when your type is any for instance)

In which case address is empty ?

Reply:
ok updated